modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -25,14 +25,19 @@ The subnets assigned to nodes and the IP addresses assigned to individual pods a
 
 While the OVN-Kubernetes network provider implements many of the capabilities present in the OpenShift SDN network provider, the configuration is not the same.
 
+* If your cluster uses any of the following OpenShift SDN capabilities, the migration to OVN-Kubernetes migrates the configuration for these features as well:
```

Comment:
4.12 start to support the SDN migration on IBM cloud platform for both UPI and IPI clusters. https://issues.redhat.com/browse/NP-575

Reply:
@weliang1, added, thanks!